### ISSUE TITLE
We want Bromite back!

### ISSUE THREAD

**Issue by ghost** [bromite/bromite#2662]
Hey,

i would love to see Bromites Comeback! Is there a chance for that?

**Comment by ghost** [comment]
Yes, it would be nice to hear from the developer. He ignored all questions about it and development was simply stopped at that time, there was never an announcement. We don't know what happened, but we should respect the capacity and possibilities of one person. Bromite was developed by one person. And at least the developer updated the Bromite URL a few weeks ago. Why don't you use Cromite? I think even if @csagan5 comes back, he could help us more by helping with Cromite development.

**Comment by ghost** [comment]
@AnonUserX0 Yea true and I understand that. Oh wow I did not know that with the URL. I use Cromite. Yea.

**Comment by HarriBuh** [comment]
Nobody needs Bromite back, because Cromite has taken this place and it's unlimited times better than Bromite ever was.

**Comment by DeadSecTr** [comment]
For what?

**Comment by ioc2e3** [comment]
> ... , he could help us more by helping with Cromite development.

Totally agree

.

**Comment by soldiaz** [comment]
Bromite can be used to watch waipu in browser, Cromite refuses and youre forced to download app
